.github/workflows/build-manylinux-container-images.yml

```
@@ -22,7 +22,7 @@ jobs:
   build:
     runs-on: ${{ matrix.IMAGE.HOST_OS || 'ubuntu-latest' }}
 
-    timeout-minutes: 45
+    timeout-minutes: 60
```

Let's try this. It looks like the slowest runs are close to 45m so 50m should be enough. This is meant to be tight in the interest of responsiveness and saving compute.
```suggestion
    timeout-minutes: 50
```

---

Trying with 50. Lets see.